src/components/shared/CopyableAddressV2/CopyableAddressV2.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback } from 'react';
+import clsx from 'clsx';
+import { splitAddress } from '~utils/strings';
+
+import { Address } from '~types';
+
+import MaskedAddress from '~shared/MaskedAddress';
+import useClipboardCopy from '~hooks/useClipboardCopy';
+import Icon from '~shared/Icon';
+// Note that we use CSS here because MaskedAddress css file will override font-normal by tailwind due to injection order
```

Comment:
We need to fix that, we have to be able to override existing CSS with tailwind classes

Reply:
You could re-write the entire MaskedAddress component using tailwind I suppose? Doesn't matter if you break "old world" components